### PR TITLE
Update demo pages to show perf metrics

### PR DESCRIPTION
**Pull request by gregjopa**
Adds DOMContentLoaded and Page Load time metrics to the demo pages:

Screen Shot 2021-11-04 at 4 42 24 PM

It would be great to figure out how to measure when first paint happens to compare that metric across component.

### CODE REVIEW

**Review by wsbrunson — APPROVED** [review]
looks sweet! two things

1. Another good project might be figuring out how we can share these tables across any page
2. We could look into https://web.dev/vitals/ for reporting more metrics

**Review by westeezy — APPROVED** [review]
nice!

**Comment by gregjopa** [comment]
> looks sweet! two things
> 
>     1. Another good project might be figuring out how we can share these tables across any page

Great suggestion! This inspired me to dive into web components and I created the `<performance-table>` component to share this logic. Web Components are pretty awesome! It will be interesting to see the browser support for them in BrowserStack.

>     2. We could look into https://web.dev/vitals/ for reporting more metrics

I need to dig into web vitals more. The library looks really useful. Great future PR.

File: docs/components/buttons.html [hunk 1]
```diff
@@ -54,37 +31,18 @@ <h1>PayPal JS SDK Buttons Test Page</h1>
...
             }
 
             function onApproveCallback(data, actions) {
-                return actions.order.capture().then((details) => {
-                    setInnerHTML(
-                        "[data-testid='order-approve-message']",
-                        `Transaction completed by ${details.payer.name.given_name}`
-                    );
+                return actions.order.capture().then((orderDetails) => {
+                    console.log({ orderDetails });
```
Comment (gregjopa) [inline docs/components/buttons.html:41]: We aren't testing the order id or the order confirmation message currently so removing this for now. The order ID is in the popup url so we can assert against that pretty easy.

**Pull request merged**